ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```diff
@@ -37,6 +37,16 @@ if [[ -n ${MCE} ]] ; then
   fi
 fi
 
+function SUPPORT_NP_SKEW() {
+  curl -L "https://github.com/mikefarah/yq/releases/download/v4.31.2/yq_linux_$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/')" -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
This step is unstable. Suggest to move this line into dockerfiles and integrate the yq into your images.

Reply:
We will need to take github api rate limit into account.

Reply:
Seem we have lot places to use `yq` as this approach, could we change it centrally later?

Reply:
Ack